modules/kubernetes/kubernetes.nu

```diff
@@ -70,7 +70,8 @@ export def kk [p: path] {
 ### ctx
 export def "kube-config" [] {
     let file = if ($env.KUBECONFIG? | is-empty) { $"($env.HOME)/.kube/config" } else { $env.KUBECONFIG }
-    { path: $file, data: (cat $file | from yaml)}
+    # :FIXME: the order of fields loaded via `from yaml` is unstable
+    { path: $file, data: (cat $file | yq -o=json | from json)}
```

Comment:
I'm not a fan of using third-party tools in these scripts to parse data

Reply:
me too. 
But after importing new config using `from yaml`, maybe half of the lines in my main config file change. I can't simply use `diff` to verify that the configuration file was updated correctly.

Reply:
ok, I will reverse this.